doc/tutorial/introduction/index.md

```diff
@@ -164,7 +151,7 @@ algorithm which then executes all alignments in parallel and stores the results
 objects. The second argument to seqan3::align_pairwise is the *configuration* which allows you to specify
 a lot of parameters for the alignment computation, for instance score functions, banded alignment and whether
 you wish to compute a traceback or not. The configurations have their own namespace seqan3::align_cfg and can
-be piped for building combinations. Check the alignment tutorial if you want to learn more.
+be ORed for building combinations. Check out the alignment tutorial if you want to learn more.
```

Comment:
```suggestion
be combined via the logical OR operator (`|`) for building combinations. Check out the alignment tutorial if you want to learn more.
```
`ORed` sounds weird..

Reply:
https://dict.leo.org/forum/viewUnsolvedquery.php?idThread=157080&idForum=1&lang=en&lp=ende

Reply:
the term "piping" refers to the shell and extra-ordinarily also combining views, but it's not general for use with `|` IIANM.

Reply:
I'll agree with @smehringer

Reply:
yes, I agree to not call it piping, but not ORed either, because people may not know what to make of this. That's why I proposed the long version :D